packages/core/src/store/feed-aggregation-store.ts

```diff
@@ -77,7 +81,7 @@ export class FeedAggregationStore extends ObjectStore<number, StreamID> {
     }
   }
 
-  streamIDs(gt?: string): ReadableStream<StreamID> {
+  streamIDs(gt?: string): ReadableStream<[string, StreamID]> {
```

Comment:
nitpick: I'd prefer to use a type with named fields for the result rather than a tuple where callers need to know what the values at different indices represent.  But not a big deal, if you want to leave it as-is can we at least add some comments about the structure of the return value?

Reply:
Okay.